vast.go

```diff
@@ -13,7 +13,11 @@ type VAST struct {
 	Ads []Ad `xml:"Ad"`
 	// Contains a URI to a tracking resource that the video player should request
 	// upon receiving a “no ad” response
-	Errors []string `xml:"Error"`
+	Errors []CDATA `xml:"Error"`
```

Comment:
You missed the `Errors` in the InLine struct https://github.com/MadHive/vast/blob/4a7b87d90d554b07cc3bad8bfcf43718f10c7256/vast.go#L69